Gems/Pointcloud/Code/Source/Tools/Components/PointcloudEditorComponent.cpp

```diff
@@ -86,12 +86,14 @@ namespace Pointcloud
             });
         AzToolsFramework::EditorEntityInfoNotificationBus::Handler::BusConnect();
         AZ::TransformNotificationBus::Handler::BusConnect(GetEntityId());
+        PointcloudEditorComponentConfigurationBus::Handler::BusConnect(GetEntityId());
     }
 
     void PointcloudEditorComponent::Deactivate()
     {
         AZ::TransformNotificationBus::Handler::BusDisconnect();
         AzToolsFramework::EditorEntityInfoNotificationBus::Handler::BusDisconnect();
+        PointcloudEditorComponentConfigurationBus::Handler::BusDisconnect();
```

Comment:
Buses should be disconnected in reverse order to connection.

Reply:
Done